conf/defaults.ini

```diff
@@ -264,6 +264,9 @@ password_hint = password
 # Default UI theme ("dark" or "light")
 default_theme = dark
 
+# Default timezone
+default_timezone = browser
+
```

Comment:
can you move this to the new [date_formats]  section added in https://github.com/grafana/grafana/pull/27216

Reply:
yes, need to add what timezone format is supported and have validation

Reply:
Move the setting to date_formats settings and added validation. I am wondering if this is the right section, because timezone is not really a date format. I don't know of a better section thou.